axiom/nr_distributed_trace.c

```diff
@@ -1256,7 +1272,7 @@ char* nr_distributed_trace_create_w3c_traceparent_header(const char* trace_id,
    * 0's.
    */
   tmp = nr_string_to_lowercase(trace_id);
-  padding = 32 - nr_strlen(tmp);
+  padding = NR_TRACE_ID_SIZE - nr_strlen(tmp);
```

Comment:
This should really say `NR_W3C_TRACE_ID_SIZE`. It just so happens that the values for `NR_W3C_TRACE_ID_SIZE` is 32 which is the size of the padded internal trace id. But if either of the specs changes, it shouldn't affect the other place.